### PR TITLE
Provide default support to es6 files.

### PR DESCRIPTION
**Pull request by emphaticsunshine**
Provide default support to es6 files as they are supported by js-beautify.

### CODE REVIEW

**Comment by vkadam** [comment]
Will you please provide config snippet, how you are using new attribute from your grunt file?

**Comment by emphaticsunshine** [comment]
So now I can have glob pattern like this:

`!tests/uitests/lib/**/*.{js,es6}`

Earlier it was:
``!tests/uitests/lib/**/*.js`

Because it didn't understand es6 file type.

**Comment by vkadam** [comment]
Mapping different extension with core type (js/css/html) is already supported. Can you try it
https://github.com/vkadam/grunt-jsbeautifier#3-beautify-files-other-than-js-json-css--html

```
js: {
    fileTypes: [".es6"]
}
```

Above config will format js, json and es6

If you would like to make es6 as default you need code change at https://github.com/vkadam/grunt-jsbeautifier/blob/master/lib/jsbeautifier.js#L153

**Comment by emphaticsunshine** [comment]
I already fixed that in my existing project by adding `es6` to fileTypes, but I think it should be supported by default without adding that in config. That's what this pull request is about.

**Comment by emphaticsunshine** [comment]
Made changes as you guided. Does it make sense?

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "node": ">=0.8"
     },
     "dependencies": {
-        "grunt": ">=0.4.1",
+        "grunt": "^0.4.1",
```
Comment (emphaticsunshine) [inline package.json]: Has to be 0.x.x

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
         "grunt-contrib-jshint": "^0.11.0",
         "grunt-contrib-nodeunit": "^0.4.1",
         "grunt-dev-update": "^1.0.2",
+        "mocha": ">1.20.0",
```
Comment (emphaticsunshine) [inline package.json]: added peerDependency for grunt-mocha-test

**Comment by vkadam** [comment]
v0.2.11 published

**Comment by emphaticsunshine** [comment]
Thanks @vkadam

**Pull request merged**